alembic/versions/ac2686b45596_refactor_resource_and_roomresource_.py

```diff
@@ -0,0 +1,30 @@
+"""Refactor Resource and RoomResource Tables
+
+Revision ID: ac2686b45596
+Revises: f960459f3067
+Create Date: 2019-05-03 17:11:29.139367
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ac2686b45596'
+down_revision = 'f960459f3067'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('resources', 'quantity')
```

Comment:
Why are we dropping the quantity field of the resources table? @KMaina cc @joshuaocero